dot/parachain/types/types.go

```diff
@@ -26,6 +26,10 @@ type ValidatorID [sr25519.PublicKeyLength]byte
 // BlockNumber The block number type.
 type BlockNumber uint32
 
+func (b BlockNumber) Encode() ([]byte, error) {
```

Comment:
why does block number need to be custom encoded as a pointer?

Reply:
Good catch! It was only being used by tests. I've removed it.